Kickstarter-iOS/Views/Cells/DiscoveryLightsOnEditorialCell.swift

```diff
@@ -0,0 +1,134 @@
+import Foundation
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class DiscoveryLightsOnEditorialCell: UITableViewCell, ValueCell {
```

Comment:
Given that this is so similar to `DiscoveryEditorialCell`, which is named in such a way that it's not specific to a particular editorial, could we not reuse that cell for this?